packages/react-storage/src/components/StorageBrowser/Views/Controls/Download.tsx

```diff
@@ -27,7 +49,7 @@ const BLOCK_NAME = `${CLASS_BASE}__download`;
 export interface DownloadControl<
   T extends StorageBrowserElements = StorageBrowserElements,
 > {
-  (): React.JSX.Element;
+  (props: { imgKey: string }): React.JSX.Element;
```

Comment:
Downloads aren't limited to images right 😅

Reply:
Lol yes good point